src/System.Text.Json/src/System/Text/Json/Node/JsonString.cs

```diff
@@ -91,7 +91,7 @@ public string Value
         /// <exception cref="FormatException">
         ///   Text value of this instance is not in an ISO 8601 defined DateTimeOffset format.
         /// </exception>
-        public DateTimeOffset GetDateTimeOffset() => DateTimeOffset.ParseExact(_value, "s", CultureInfo.InvariantCulture);
+        public DateTimeOffset GetDateTimeOffset() => JsonHelpers.TryParseAsISO(_value.AsSpan(), out DateTimeOffset value) ? value : throw new FormatException(SR.FormatDateTimeOffset);
```

Comment:
Same `.AsSpan()` question for all of these.  I'd expect the implicit conversion would work fine.